sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -371,3 +371,27 @@
         'WriteToTFRecord': 'beam:schematransform:org.apache.beam:tfrecord_write:v1'
       config:
         gradle_target: 'sdks:java:io:expansion-service:shadowJar'
+
+#BigTable
+- type: renaming
+  transforms:
+    'ReadFromBigTable': 'ReadFromBigTable'
+    'WriteToBigTable': 'WriteToBigTable'
+  config:
+    mappings:
+      'ReadFromBigTable':
+        project: 'project_id'
+        instance: 'instance_id'
+        table: 'table_id'
```

Comment:
let's not expose ReadFromBigtable just yet

Reply:
Let's remove it too. It should not be exposed to users

Do you mean IT tests are failing without it? How come?